contracts/SpokePool.sol

```diff
@@ -467,18 +471,21 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
         address destinationToken,
         uint256 amount,
         uint256 originChainId,
+        uint256 destinationChainId,
```

Comment:
Maybe we should omit this to save calldata and _assume_ that the current chainId is the one they intended? Thoughts?

Reply:
Yup done